partner_second_lastname/tests/test_multiple_name.py

```diff
@@ -0,0 +1,31 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestRecalculateNames(TransactionCase):
+    def test_recalculate_names(self):
+        firstname = "Xavier De Jesús"
+        lastname = "Payen"
+        lastname2 = "Sandoval"
+        partner = self.env["res.partner"].create(
+            {
+                "firstname": firstname,
+                "lastname": lastname,
+                "lastname2": lastname2,
+            }
+        )
+        config_settings = self.env["res.config.settings"].create({})
+        order_names_dict = {
+            "first_last": f"{firstname} {lastname} {lastname2}",
+            "last_first": f"{lastname} {lastname2} {firstname}",
+            "last_first_comma": f"{lastname} {lastname2}, {firstname}",
+        }
+        for order, expected_name in order_names_dict.items():
```

Comment:
We should avoid loops inside tests, because tests should be as deterministic as possible.

If you move code inside loop to a different method and you call that method three times, you will end up with even less lines than currently.